src/Faker/Name.cs

```diff
@@ -46,6 +46,15 @@ public static class Name
 		/// <returns>The randomly created first name.</returns>
 		public static string First()
 		{
+			return Resources.Name.First.Split(Config.SEPARATOR).Random().Trim();
+		}
+
+		/// <summary>
+		///   Creates a random first name, using a more aggressive caching strategy.
+		/// </summary>
+		/// <returns>The randomly created first name.</returns>
+		public static string FirstCached()
```

Comment:
1. Is not this kind of caching fully belonging to the internals? Why could it be breaking?
2. High unlikely, yes. Resource are static, compile-time information. So, from the user point of view they are read-only. Why could one prefer a non-cached value?